packages/material-ui/src/Popper/Popper.test.js

```diff
@@ -141,16 +143,21 @@ describe('<Popper />', () => {
   });
 
   describe('prop: popperOptions', () => {
-    it('should pass all popperOptions to popperjs', (done) => {
-      const popperOptions = {
-        onCreate: (data) => {
-          data.instance.update({ placement: 'left' });
-        },
-        onUpdate: () => {
-          done();
-        },
-      };
-      render(<Popper {...defaultProps} popperOptions={popperOptions} placement="top" open />);
+    it('should pass all popperOptions to popperjs', () => {
+      const popperRef = React.createRef();
+      const { setProps } = render(
+        <Popper {...defaultProps} popperRef={popperRef} placement="top" open />,
+      );
+
+      act(() => {
```

Comment:
You sure that is needed? I think setProps calls testing-library#render which is already wrapped.